apps/web/pages/booking/[uid].tsx

```diff
@@ -485,7 +485,7 @@ export default function Success(props: SuccessProps) {
                       </>
                     )}
                     <div className="font-medium">{t("what")}</div>
-                    <div className="col-span-2 mb-6 last:mb-0">{eventName}</div>
+                    <div className="col-span-2 mb-6 last:mb-0">{props.bookingInfo.title}</div>
```

Comment:
@G3root can you comment why nameless was coming and how switching from eventName to props.bookingInfo.title fixed it?

Also, is it something that recently regressed or like this always.

Reply:
previously event name has been derived from [here](https://github.com/calcom/cal.com/blob/1bf50e09cdeb4eb06683db080e9bd61abe991671/apps/web/pages/booking/%5Buid%5D.tsx#LL217-L220C20), in the event of cancelled booking , attendees are empty so it fallbacks to `Nameless`. using the title of the booking seems reliable here.

i think it is recently regressed.